starfish/core/spots/FindSpots/local_max_peak_finder.py

```diff
@@ -309,7 +317,7 @@ def run(
         n_processes : Optional[int] = None,
             Number of processes to devote to spot finding.
         """
-        spot_finding_method = partial(self.image_to_spots, *args, **kwargs)
+        spot_finding_method = partial(self.image_to_spots, **self.kwargs)
```

Comment:
If *args and **kwargs are no longer used here, should they be removed from `run()` parameters?

Reply:
The issue is that the superclass's `run()` method has `*args, **kwargs`.

Reply:
I see, thanks for explaining.